### ISSUE TITLE
PAM+SSSD rejecting logins

### ISSUE THREAD

**Issue by mischief** [coreos/bugs#1265]
with a password set via coreos-cloudinit in a qemu vm:

```
This is coreos_production_qemu-1032-0-0 (Linux x86_64 4.5.2-coreos) 21:17:11
SSH host key: SHA256:8aoN3a8AI9TW6Jc1ts9uFvEXGLh5lxKuHbD51ilmSN0 (DSA)
SSH host key: SHA256:FR1cxvnV/OHEJ7dB/E5HdYT+JoXdydcC7bZCmILwFVY (ECDSA)
SSH host key: SHA256:tQkGOjiEMtBZyaX3nIotvqli5ExRKjgwsjhYDoFyuDU (ED25519)
SSH host key: SHA256:dp4Uxzx9C4jeLM4zucCx/hVGsJfqJO7/iHmGR0OCX6M (RSA)
eth0: 10.0.2.15 fe80::5054:ff:fe12:3456

coreos_production_qemu-1032-0-0 login: core
Password: 

Authentication service cannot retrieve authentication info
```

setting `coreos.autologin=` on the kernel commandline produces a similar failure on the console.

ssh also does not work, with a configured password or ssh key.

**Comment by mischief** [comment]
cc @mjg59

**Comment by mjg59** [comment]
https://github.com/coreos/coreos-overlay/pull/1946

**Comment by mischief** [comment]
fixed by coreos/coreos-overlay#1946
